sw/device/benchmarks/coremark/top_earlgrey/ee_printf.c

```diff
@@ -480,7 +480,12 @@ static int ee_vsprintf(char *buf, const char *fmt, va_list args) {
 
   for (str = buf; *fmt; fmt++) {
     if (*fmt != '%') {
-      *str++ = *fmt;
+      if(*fmt == '\n') {
```

Comment:
nit: `if (`

Please run this file through clang-format (all files except for the vendor files)